projects/commudle-admin/src/app/feature-modules/user-chats/components/chats-container/chats-container.component.ts

```diff
@@ -0,0 +1,115 @@
+import {ChangeDetectionStrategy, Component, OnDestroy, OnInit} from '@angular/core';
+import {IDiscussionFollower} from '../../../../../../../shared-models/discussion-follower.model';
+import {ICurrentUser} from '../../../../../../../shared-models/current_user.model';
+import {SDiscussionsService} from '../../../../../../../shared-components/services/s-discussions.service';
+import {LibAuthwatchService} from '../../../../../../../shared-services/lib-authwatch.service';
+import {UserChatsService} from '../../services/user-chats.service';
+import {UserChatMessagesChannel} from '../../services/websockets/user-chat-messages.channel';
```

Comment:
I think these reference paths need to be fixed, let's make them absolute paths

Reply:
This is done.